### ISSUE TITLE
Add KATL

### ISSUE THREAD

**Issue by aiannuzzi99** [openscope/openscope#500]
Hi, I was wondering if you were taking requests about which airports to add to the openscope ATC sim.  I would really like to see KATL appear on that list, after how professional the KORD and KDFW sectors look, I think ATL would look fantastic.  If you do decide the implement ATL I am a controller at ATL on VATSIM and I could proably provide you with some sector files if needed or airspace diagrams.

**Comment by erikquinn** [comment]
[https://youtu.be/_FTyXnlZAuk](https://youtu.be/_FTyXnlZAuk)

It's actually already in there, but I guess it's broken and that's why you're not seeing it in the list. Not sure what the issue is with it, but we could definitely use your expertise with documentation on A80 airspace and such!

Feel free to join us on [slack](http://slack.openscope.co)!

**Comment by aiannuzzi99** [comment]
So is there any way to make it not hidden or is it officially broken

**Comment by indianbhaji** [comment]
It's actually not in the game at all it appears @erikquinn @aiannuzzi99

**Comment by Fechulo** [comment]
I'm currently on top of a mountain and connection is nowhere to be found so I won't be able to communicate for the next week.

Regarding KATL, I worked on the airport a while ago but it was when @n8rzz was rewriting the entire game so no airport changes were being merged. So I never got around to creating a PR for it. The video was recorded on my fork of the sim not openscope.co.

KATL is not the only airport I worked on during that time so I've got a fairly long list of airports that I want to add. However, I want to prioritize updating the already existing airports and then work on new ones.
